Modules/EMCAL/src/RawErrorCheckAll.cxx

```diff
@@ -0,0 +1,129 @@
+#include "QualityControl/MonitorObject.h"
+#include "EMCAL/RawErrorCheckAll.h"
+#include "QualityControl/QcInfoLogger.h"
+#include "QualityControl/Quality.h"
+
+// ROOT
+#include <TH1.h>
+#include <TH2.h>
+#include <TPaveText.h>
+#include <TLatex.h>
+#include <TList.h>
+#include <TRobustEstimator.h>
+#include <ROOT/TSeq.hxx>
+#include <iostream>
+#include <vector>
+#include <TCanvas.h>
+#include <string>
+#include <TGraph.h>
+#include <queue>
+
+using namespace std;
+
+namespace o2::quality_control_modules::emcal
+{
+void RawErrorCheckAll::configure()
+{
+    
+    // configure threshold-based checkers
+    auto nBadThreshold = mCustomParameters.find("BadThreshold");
+    if (nBadThreshold != mCustomParameters.end()) {
+        try {
+            mBadThreshold = std::stod(nBadThreshold->second);
+        } catch (std::exception& e) {
+            ILOG(Error, Support) << fmt::format("Value {} not a double", nBadThreshold->second.data()) << ENDM;
```

Comment:
For c++ streams like `ILOG` you don't need any formatting library, as this is handled by the iostream library. In any case, with c++20's std::format the format library anyhow kind of becomes obsolete.